node/service.json

```diff
@@ -2,7 +2,7 @@
   "stack": "nodejs",
   "memory": 256,
   "ttl": 60,
-  "timeout": 30,
+  "timeout": 45,
```

Comment:
why are we changing the timeout? and why 45? also, the changelog says we are increasing it to 60.

Reply:
I was getting a timeout error on this call, I don't know if this happens in prd too. I kept trying until I found a number that the timeout stopped. But if you have another suggestion, we can remove this one.

![Screenshot 2024-09-09 at 14 43 22](https://github.com/user-attachments/assets/bcae69a3-d03c-45ae-a701-8c32780a2313)